src/encoded/schemas/tissue_sample.json

```diff
@@ -69,22 +77,36 @@
         "schema_version": {
             "default": "1"
         },
-        "tissue_location": {
-            "title": "Tissue Location",
-            "description": "Original location of sample within source tissue",
+        "submitted_id": {
+            "pattern": "^[A-Z0-9]{3,}_TISSUE-SAMPLE_[A-Z0-9-_.]{4,}$"
+        },
+        "category": {
```

Comment:
Why did you add "category" mixin that is array type and then override - I think you need to remove the mixin?